packages/fxa-auth-server/config/index.ts

```diff
@@ -1923,6 +1923,14 @@ const conf = convict({
     env: 'SYNC_TOKENSERVER_URL',
     format: 'url',
   },
+  guardian: {
```

Comment:
I don't know if other RPs might need this as well for their app support. Maybe call it `support`?